README.md

```diff
@@ -54,6 +54,9 @@ Community discussions happen primarily on the [dev mailing list][dev-list] or on
 
 Iceberg is built using Gradle 5.2.1.
 
+* To invoke a build and run tests, run: ./gradlew build
+* To skip tests, run: ./gradlew build -x test
```

Comment:
This is cool.

Reply:
Next to skipping tests we could also document how to skip checkstyle violations (it usually helps lower the adoption barrier for code changes) `./gradlew clean build -x test -x check`. Wdyt?

Reply:
@fbocse Sounds good. Adding it

Reply:
Checkstyle violations will be caught in CI testing and block PRs, so I'm not sure that excluding that check is a good idea. Maybe just note that this is okay for development, but will cause PRs to fail validation.